evm_arithmetization/src/cpu/kernel/tests/transaction_parsing/parse_type_1_txn.rs

```diff
@@ -0,0 +1,120 @@
+use anyhow::Result;
+use ethereum_types::U256;
+use hex_literal::hex;
+use plonky2::field::goldilocks_field::GoldilocksField as F;
+use NormalizedTxnField::*;
+
+use crate::cpu::kernel::aggregator::KERNEL;
+use crate::cpu::kernel::constants::txn_fields::NormalizedTxnField;
+use crate::cpu::kernel::interpreter::Interpreter;
+use crate::memory::segments::Segment;
+
+#[test]
+fn process_type_1_txn() -> Result<()> {
+    let process_type_1_txn = KERNEL.global_labels["process_type_1_txn"];
+    let process_normalized_txn = KERNEL.global_labels["process_normalized_txn"];
+
+    let retaddr = 0xDEADBEEFu32.into();
+    const INITIAL_TXN_RLP_ADDR: usize = Segment::RlpRaw as usize + 1;
+    let mut interpreter: Interpreter<F> = Interpreter::new(
+        process_type_1_txn,
+        vec![retaddr, INITIAL_TXN_RLP_ADDR.into()],
+        None,
+    );
+
+    // When we reach process_normalized_txn, we're done with parsing and
+    // normalizing. Processing normalized transactions is outside the scope of
+    // this test.
+    interpreter.halt_offsets.push(process_normalized_txn);
```

Comment:
We could wrap all of these initializations in some `prepare_interpreter(start_label: String)` method defined in the `transaction_parsing` module, as done for other Kernel tests to reduce code duplication.

Reply:
We could even do the verification in a common method for most fields FWIW, I think only the signature fields / specific type-fields are different? But it may be a bit messy if we mix verification checks.

Reply:
For field verification, since the values are entered manually anyway, I don't think it changes much, and I find it clear enough to leave it as is.